packages/lab/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -84,6 +84,11 @@ export const DataDriven: StoryObj<HvDashboardProps> = {
       },
     },
     eyes: { include: false },
+    a11y: {
+      config: {
+        rules: [{ id: "aria-prohibited-attr", enabled: false }],
```

Comment:
Can we also add the color contrast rule (`{ id: "color-contrast", enabled: false }`)? 🙏 We don't have a11y issues with the dawn color mode but we have it with wicked.

Reply:
Sure, PR updated.